federatedscope/mf/model/model.py

```diff
@@ -15,47 +14,34 @@ class BasicMFNet(Module):
     """
     def __init__(self, num_user, num_item, num_hidden):
         super(BasicMFNet, self).__init__()
-
-        self.embed_user = Parameter(
-            torch.normal(mean=0,
-                         std=0.1,
-                         size=(num_user, num_hidden),
-                         requires_grad=True,
-                         dtype=torch.float32))
-        self.register_parameter('embed_user', self.embed_user)
-        self.embed_item = Parameter(
-            torch.normal(mean=0,
-                         std=0.1,
-                         size=(num_item, num_hidden),
-                         requires_grad=True,
-                         dtype=torch.float32))
-        self.register_parameter('embed_item', self.embed_item)
+        self.num_user, self.num_item = num_user, num_item
+        self.embed_user = torch.nn.Embedding(num_user, num_hidden, sparse=True)
```

Comment:
What's the difference between `torch.nn.Embedding` and `torch.normal`, and is `torch.nn.Embedding` much faster than `torch.normal`?